timesketch/frontend/src/components/Common/DynamicTableDisplay.vue

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright 2021 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <section class="section">
+      <div class="container is-fluid">
+        <div class="card" style="min-height:160px;">
```

Comment:
Do we want to include the wrapping element/style?

To make the component a bit easier to use in different contexts you could just render the table and let the caller (view) add it to whatever element (card, section etc) it wants.

WDYT?

Reply:
My idea here was to encourage some consistency on the UI. The wrapping is fairly minimal here; I think most users of that component would be OK with it.